examples/scenarioDeployingSolarArrays.py

```diff
@@ -0,0 +1,724 @@
+#
+# ISC License
+#
+# Copyright (c) 2024, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
+#
+# Permission to use, copy, modify, and/or distribute this software for any
+# purpose with or without fee is hereby granted, provided that the above
+# copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+# WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+# MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+# ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+# WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+# ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+# OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+#
+
+r"""
+Overview
+--------
+
+This scenario demonstrates the multi-body prescribed motion dynamics capability of the
+:ref:`prescribedMotionStateEffector` module through an illustrative rotational solar array deployment scenario.
+The spacecraft in this example consists of a rigid hub and two symmetrical solar arrays. Each array is modeled as a
+collection of 10 prescribed motion elements. Therefore, 20 instances of the :ref:`prescribedMotionStateEffector` module
+are required to simulate the array deployments.
+
+Note that in order to simulate hub-relative prescribed motion, kinematic profiler modules must be connected to the
+prescribed motion state effector modules. The kinematic profiler modules specify the translational and rotational
+states of each prescribed sub-component relative to the spacecraft hub and write the sub-component states at each time
+step to the prescribed motions state effector modules using the Basilisk messaging system. Currently two kinematic
+profiler modules exist in Basilisk that can be used to simulate prescribed motion. The first
+:ref:`prescribedLinearTranslation` module prescribes linear translational motion of a prescribed sub-component
+relative to the hub; while the second :ref:`prescribed1DOFRotation` module prescribes 1 DOF rotational motion relative
```

Comment:
This module `prescribed1DOFRotation` does not exist? Please update label.

Reply:
Done